vista-fhir-query-tests/src/test/java/gov/va/api/health/vistafhirquery/tests/r4/OrganizationIT.java

```
@@ -0,0 +1,28 @@
+package gov.va.api.health.vistafhirquery.tests.r4;
+
+import static gov.va.api.health.sentinel.EnvironmentAssumptions.assumeEnvironmentNotIn;
+
+import gov.va.api.health.fhir.testsupport.ResourceVerifier;
+import gov.va.api.health.r4.api.resources.OperationOutcome;
+import gov.va.api.health.r4.api.resources.Organization;
+import gov.va.api.health.sentinel.Environment;
+import gov.va.api.health.vistafhirquery.tests.TestIds;
+import gov.va.api.health.vistafhirquery.tests.VistaFhirQueryResourceVerifier;
+import lombok.experimental.Delegate;
+import lombok.extern.slf4j.Slf4j;
+import org.junit.jupiter.api.Test;
+
+@Slf4j
+public class OrganizationIT {
+  private final TestIds testIds = VistaFhirQueryResourceVerifier.ids();
+  @Delegate private final ResourceVerifier verifier = VistaFhirQueryResourceVerifier.r4();
+
+  @Test
+  void read() {
+    assumeEnvironmentNotIn(Environment.STAGING, Environment.PROD);
```

She's gonna fail hard in s-lab/lab as well.